processing/data_collection/gazette/spiders/pr_foz_do_iguacu.py

```diff
@@ -0,0 +1,71 @@
+import datetime as dt
+
+import scrapy
+from scrapy.http import FormRequest
+
+from gazette.items import Gazette
+
+
+class PrFozDoIguacuSpider(scrapy.Spider):
+    MUNICIPALITY_ID = '4108304'
+    name = 'pr_foz_do_iguacu'
+    allowed_domains = ['pmfi.pr.gov.br']
+    start_urls = ['http://www.pmfi.pr.gov.br/utilidades/diario/index.xhtml']
+    AVAILABLE_FROM = dt.date(2015, 1, 1)  # actually September/2007
+
+    def parse(self, response):
+        """
+        @url http://www.pmfi.pr.gov.br/utilidades/diario/index.xhtml
+        @returns requests 1 1
+        """
+        selector = '(//span[@class="ui-paginator-current"])[1]/text()'
+        paginator_text = response.xpath(selector).extract_first()
+        quantity_of_documents = int(paginator_text[1:-1].split()[-1]) + 1
+
+        data = {
+            'formDiarioOficialView': 'formDiarioOficialView',
+            'formDiarioOficialView:dtdiario': 'formDiarioOficialView:dtdiario',
+            'formDiarioOficialView:dtdiario_encodeFeature': 'true',
+            'formDiarioOficialView:dtdiario_first': '0',
+            'formDiarioOficialView:dtdiario_pagination': 'false',
+            'formDiarioOficialView:dtdiario_rows': str(quantity_of_documents),
+        }
+        return FormRequest(response.url, formdata=data, callback=self.parse_items)
+
+    def parse_items(self, response):
+        """
+        @url http://www.pmfi.pr.gov.br/utilidades/diario/index.xhtml
+        @returns items 10 10
+        """
+        base_url = 'http://www.pmfi.pr.gov.br{}'
```

Comment:
What about moving this to a constant? `BASE_URL`.

Reply:
Done! tks